cognite/client/_api/assets.py

```diff
@@ -558,7 +558,7 @@ def create_hierarchy(
         Args:
             assets (Sequence[Asset | AssetWrite] | AssetHierarchy): List of assets to create or an instance of AssetHierarchy.
             upsert (bool): If used, already existing assets will be updated instead of an exception being raised. You may control how updates are applied with the 'upsert_mode' argument.
-            upsert_mode (Literal["patch", "replace"]): Only applicable with upsert. Pass 'patch' to only update fields with non-null values (default), or 'replace' to do full updates (unset fields become null or empty).
+            upsert_mode (Literal['patch', 'replace']): Only applicable with upsert. Pass 'patch' to only update fields with non-null values (default), or 'replace' to do full updates (unset fields become null or empty).
```

Comment:
Run pre-commit with 3.8 to comply with CI checks

Reply:
Fixed in [a547417](https://github.com/cognitedata/cognite-sdk-python/pull/1749/commits/a5474175c09b23758579e7e9eddd2ba2ea4c84fc)